src/domains/optimism/OptimismDomainHost.sol

```diff
@@ -136,18 +132,18 @@ contract OptimismDomainHost is DomainHost {
         );
     }
 
-    function tell(uint256 _lid, uint256 value) external guestOnly {
-        _tell(_lid, value);
+    function tell(uint256 _lid, uint256 debt) external guestOnly {
+        _tell(_lid, debt);
     }
 
     function exit(address usr, uint256 wad) external {
         exit(usr, wad, glExit);
     }
     function exit(address usr, uint256 wad, uint32 gasLimit) public {
-        _exit(usr, wad);
+        (uint256 claim) = _exit(usr, wad);
```

Comment:
nit: don't need the brackets around `claim`.

Reply:
Yeah but as I thing I saw in other functions it was being used this way that's why I kept the same styling but I might remove it from everywhere.